### PR TITLE
Hooking up profile and password forms

### PR DESCRIPTION
**Pull request by jasonrhodes**

### CODE REVIEW

File: src/pages/settings/Profile/UpdatePasswordForm.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <TextField {...rest} {...input} error={error} />
+);
+
+export class UpdatePasswordForm extends Component {
+  renderError () {
+    const { passwordError } = this.props;
+    if (!passwordError) return null;
+
+    return (
+      <div class='text-danger'>
```
Comment (jonambas) [inline src/pages/settings/Profile/UpdatePasswordForm.js]: className

although this css class doesn't

File: src/reducers/index.js [hunk 2]
```diff
@@ -10,5 +10,16 @@ export default combineReducers({
   account,
   currentUser,
   metrics,
-  form: reduxFormReducer
+  form: reduxFormReducer.plugin({
+    updatePassword: (state, action) => {
```
Comment (jonambas) [inline src/reducers/index.js]: What is this for?

File: src/pages/settings/Profile/UpdateProfileForm.js [hunk 3]
```diff
@@ -25,30 +27,28 @@ export class NameForm extends Component {
...
-          name='lastName'
+          name='last_name'
           id='lirstName'
           label='Last Name'
           component={TextFieldWrapper}
         />
 
-        <Button submit>Update Profile</Button>
+        <Button submit>
+          { updating ? 'Updating Profile...' : 'Update Profile' }
```
Comment (jonambas) [inline src/pages/settings/Profile/UpdateProfileForm.js]: you can also use form.profileName.submitting i think, although it doesnt look like the redux-form submit action is being triggered

File: src/reducers/index.js (hunk 2 above)
Comment (jasonrhodes) [inline src/reducers/index.js]: This is how i'm clearing the form after any submit ... can't figure out a cleaner way yet

File: src/pages/settings/Profile/UpdateProfileForm.js (hunk 3 above)
Comment (jasonrhodes) [inline src/pages/settings/Profile/UpdateProfileForm.js]: Yeah this.props.submitting works -- i think i have it working, pushing in a minute

File: src/reducers/index.js (hunk 2 above)
Comment (jonambas) [inline src/reducers/index.js]: looks like this.props.reset() is available in the form

but otherwise :shipit:

**Comment by jonambas** [comment]
👍  looks good

**Comment by jasonrhodes** [comment]
Closing this for now because tbh I don't remember the real goal here and it'll be easier to do this over later.

**Pull request closed without merging**